enhancements/cluster-logging/loki-observability.md

```diff
@@ -0,0 +1,469 @@
+the Loki Deployment. The alerts are based on metrics consumed from Prometheus into alert manager, evaluated and when
+fired, pushed to notify human operators.
+
+Loki Alerts can be categorized as follows:
+
+| Category | Description |
+|----------|-------------|
+| **Loki System** | One or more of the Loki components is misbehaving; this can be due to any of Loki or OCP system that is not working as expected |
+| **Tenant workload** | Customer (tenant) sending (writing) workload or (reading) query logs and/or acting against the system in some anomalous behavior |
+| **Storage** | The storage system that Loki works with is not working as expected. This includes internal storage and external S3 storage |
+
+Loki alerts can be of various severities:
+
+| Severity | Description |
+| -------- | ----------- |
+| **Critical** | For alerting current and impending disaster situations. These alerts page an SRE. The situation should warrant waking someone in the middle of the night. |
+| **Warning** | The vast majority of alerts should use the severity. Issues at the warning level should be addressed in a timely manner, but don't pose an immediate threat to the operation of the cluster as a whole. |
+| **Info** | Info level alerts represent situations an administrator should be aware of, but that don't necessarily require any action. Use these sparingly, and consider instead reporting this information via Kubernetes events. |
+
+See also: [Alerting Consistency](/enhancements/monitoring/alerting-consistency.md)
+#### Alerts
+
+Some alerts are based on metrics exposed by Loki and some on recording rules, which are metrics created based on other metrics.
+
+See also [Defining recording rules](https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/)
+
+Alerts summary table:
+
+| Alert | Category | Severity | Summary |
+| ----- | -------- | -------- | ----------- |
+| [LokiRequestErrors](#LokiRequestErrors) | Loki System | Critical | At least 10% of requests are responded by 5xx server errors |
+| [LokiRequestPanics](#LokiRequestPanics) | Loki System | Critical | A panic was triggered |
+| [LokiRequestLatency](#LokiRequestLatency) | Loki System | Critical | The 99th percentile is experiencing high latency (higher than 1 second) |
+| [LokiTenantRateLimitWarning](#LokiTenantRateLimitWarning) | Tenant Workload | Warning | A tenant is experiencing rate limiting |
+| [LokiCPUHighWarning](#LokiCPUHighWarning) | Loki System | Warning | Loki's CPU usage is high (above 75% for 5 minutes) |
+| [LokiCPUHighError](#LokiCPUHighError) | Loki System | Critical | Loki's CPU usage is high (above 95% for 5 minutes) |
+| [LokiMemoryHighWarning](#LokiMemoryHighWarning) | Loki System | Warning | Loki's memory usage is high (above 75% of the configured limit for 5 minutes) |
+| [LokiMemoryHighError](#LokiMemoryHighError) | Loki System | Critical | Loki's memory usage is high (above 95% of the configured limit for 5 minutes) |
```

Comment:
Same comment, both can be LokiMemoryHigh. In general, don't suffix alert names with a severity like "FooWarning", the severity is delivered as part of the alert, it shouldn't be in the name.

Reply:
Great feedback. thanks